### PR TITLE
maintenance: Update mix deps

### PR DESCRIPTION
**Pull request by jamesvl**
This a maintenance task to update internal dependencies to their latest versions.

Mix documentation will now use the latest feature, once deploy.

To view the diffs in each available update, visit:
https://hex.pm/l/kmMV8

This also a request that a new release be pushed up to hex.pm, unless there's something additional I can do to help with that.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -2,7 +2,7 @@ defmodule Ueberauth.Mixfile do
   use Mix.Project
 
   @source_url "https://github.com/ueberauth/ueberauth"
-  @version "0.10.4"
+  @version "0.10.5"
```
Comment (yordis) [inline mix.exs:5]: These are dev and test dependencies. Do we need to change the version? As far as I can tell, we do not need to cut a release for it.

Comment (jamesvl) [reply mix.exs:5]: I agree - that would be fine by me. I'll update the MR.

**Comment by yordis** [comment]
Thank you so much for helping me with the maintenance 💜 

There is a simple Credo issue that is blocking CI. Let me know if you need help.

**Comment by jamesvl** [comment]
Credo issue is also resolved now.

**Review by yordis — APPROVED** [review]

**Comment by yordis** [comment]
🚀 💜

**Comment by jamesvl** [comment]
Would you be willing to tag a new version and push the release to hex.pm?

**Pull request merged**